### PR TITLE
Add flaky-test template to track many flaky-test.

### PR DESCRIPTION
**Pull request by mattisonchao**
### Motivation

I found many flaky-test like  #3031 #3034 #3033.
Because many flaky tests are actually production code issues so I think it's a good way to add flaky-test template to track them 

### Changes

- Add flaky-test template.

### CODE REVIEW

**Comment by mattisonchao** [comment]
@merlimat @eolivelli @dlg99 @nicoloboschi PTAL :)

File: .github/ISSUE_TEMPLATE/flaky_test.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+name: Flaky test
+about: Report a flaky test failure
+title: 'Flaky-test: test_class.test_method'
+labels: ["component/test", "flaky-tests"]
```
Comment (nicoloboschi) [inline .github/ISSUE_TEMPLATE/flaky_test.md:5]: component/test doesn't exist. Please use `area/tests`. We can create the flaky-tests label @dlg99 @eolivelli

**Review by nicoloboschi — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/flaky_test.md (hunk 1 above)
Comment (mattisonchao) [reply .github/ISSUE_TEMPLATE/flaky_test.md:5]: Sure, thanks.

Comment (mattisonchao) [reply .github/ISSUE_TEMPLATE/flaky_test.md]: @nicoloboschi  PTAL : )

**Comment by mattisonchao** [comment]
rerun failure checks

**Comment by mattisonchao** [comment]
rerun failure checks

**Comment by mattisonchao** [comment]
rerun failure checks

**Comment by mattisonchao** [comment]
rerun failure checks

**Pull request merged**